lib/stripe/resources/account.rb

```diff
@@ -136,6 +136,7 @@ def deauthorize(client_id = nil, opts = {})
         client_id: client_id,
         stripe_user_id: id,
       }
+      opts = @opts.merge(Util.normalize_opts(opts))
```

Comment:
`#deauthorize` doesn't have access to `@opts`, which may contain the `client` option we need to override global configuration.